### DISCUSSION TITLE
AppxBundleManifest.xml not found in paketbasedirectory

### DISCUSSION THREAD

**Discussion by NilsRo** [LSPosed/MagiskOnWSALocal#764] [Q&A] [answered]
Hi,

I own two Dell 7706 2in1 and one 7320 Detachable and like Google Playstore integration very much! We sold all Android and iOS tablets as they are not needed anymore. But with our new 7320 Detachable I got the following error. The same package in the same manner goes through fine with both 7706. What could cause this? 

Many thanks for any tip.

```
Add-AppxPackage : Fehler bei Bereitstellung. HRESULT: 0x80073CF9, Fehler bei der Installation. Wenden Sie sich an den
Softwarehersteller. (Ausnahme von HRESULT: 0x80073CF9)
Eine Registrierungsanforderung von "AppxBundleManifest.xml" wurde abgelehnt, da die Manifestdatei nicht im
Paketstammverzeichnis vorhanden ist.
HINWEIS: Wenn Sie weitere Informationen wünschen, suchen Sie im Ereignisprotokoll nach [ActivityId]
ab58dd67-4ee5-0007-8262-5cabe54eda01, oder verwenden Sie die Befehlszeile Get-AppxLog -ActivityID
ab58dd67-4ee5-0007-8262-5cabe54eda01
In C:\WSA\Install.ps1:167 Zeichen:1
+ Add-AppxPackage -ForceApplicationShutdown -ForceUpdateFromAnyVersion  ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\WSA\AppxManifest.xml:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand
```

Cheers,
Nils

**Comment by NilsRo** [comment, marked as answer]
Solved it, there was a conflicting package installed I had to remove before. After getting all packages and deleting it the installation works fine: 

PS C:\> Get-AppxPackage -AllUsers > out.txt
PS C:\> Remove-AppxPackage -AllUsers MicrosoftCorporationII.WindowsSubsystemForAndroid_2311.40000.5.0_x64__8wekyb3d8bbwe

Perhaps helpful for others....it was triggered that an other account has installed WSA. Will open a ticket for that...

**Reply by fiftin** [reply]
Wow, thanks a lot! Works for me!
